scripts/deploy-local.js

```diff
@@ -15,18 +15,27 @@ async function deployLocal() {
 		contracts[key] = env[key].address;
 	}
 
+	const DEFAULT_ADMIN_ROLE = hre.ethers.constants.HashZero;
+	const PAUSER_ROLE = hre.ethers.utils.id("PAUSER_ROLE");
+	const TOKEN_ISSUER_ROLE = hre.ethers.utils.id("TOKEN_ISSUER_ROLE");
+
 	console.log(`Deploying bridge contract with the following addresses:`);
 	console.log(`Owner: ${adminAccount.address}`);
 	console.log(`Token Issuer: ${bridgeAccount.address}`);
 	console.log(`NCT Pool: ${env.nctPool.address}`);
 
-	const bridge = await deployBridge(bridgeAccount.address, env.nctPool.address);
+	const bridge = await deployBridge(
+		adminAccount,
+		env.nctPool.address,
+		[DEFAULT_ADMIN_ROLE, PAUSER_ROLE, TOKEN_ISSUER_ROLE],
```

Comment:
This deploys with an admin, pauser, and token issuer, but the verification args do not have a token issuer within them. Am I missing something?

Reply:
So this is for local deployment, and that's why. I still wonder why you chose to have different args vs. the other deployment script, but this comment shouldn't be blocking anything.